lib/resty/etcd.lua

```diff
@@ -61,8 +63,8 @@ function _M.new(opts)
     end
 
     opts.api_prefix = "/v2"
-    local serializer = opts.serializer or 'json'
-    opts.serializer = prequire("resty.etcd.serializers.", serializer, 'json')
+    local serializer = opts.serializer or "json"
+    opts.serializer = require_package("resty.etcd.serializers.", serializer, "json")
```

Comment:
and I think we can drop the first argument `resty.etcd.serializers.`, all right?